features/task.feature

```diff
@@ -1,31 +1,40 @@
-Feature: Verify Enterprise Contract Tekton Task
+Feature: Verify Enterprise Contract Tekton Tasks
```

Comment:
@zregvart I got the new task to work, but I'm only able to run one scenario at a time. If I run both scenarios the tests throw an error at `err = kCluster.buildCliImage(ctx)` which I found odd. Maybe each task should be its own feature? idk. just wondering your thoughts on this.

Reply:
Hmm, odd, looks okay on my end. I've pushed this to my fork and it failed twice because of a [timeout](https://github.com/zregvart/ec-cli/actions/runs/4530948515/jobs/7980501608), [timeout(2)](https://github.com/zregvart/ec-cli/actions/runs/4530948515/jobs/7980878112).

```
$ git status
On branch pr/joejstuart/545
Your branch is up to date with 'joejstuart/definition-task'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   features/task.feature

no changes added to commit (use "git add" and/or "git commit -a")
$ git diff
diff --git a/features/task.feature b/features/task.feature
index ed64646..8c5e038 100644
--- a/features/task.feature
+++ b/features/task.feature
@@ -1,38 +1,39 @@
+@focus
 Feature: Verify Enterprise Contract Tekton Tasks
   The Verify Enterprise Contract Tekton task verification against a set of golden images
 
   Background:
     Given a cluster running
 
-  # Scenario: Golden container image
-  #   Given a working namespace
-  #   Given a cluster policy with content:
-  #     ```
-  #     {
-  #       "publicKey": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAERhr8Zj4dZW67zucg8fDr11M4lmRp\nzN6SIcIjkvH39siYg1DkCoa2h2xMUZ10ecbM3/ECqvBV55YwQ2rcIEa7XQ==\n-----END PUBLIC KEY-----",
-  #       "sources": [
-  #         {
-  #           "policy": [
-  #             "github.com/hacbs-contract/ec-policies//policy"
-  #           ]
-  #         }
-  #       ],
-  #       "configuration": {
-  #         "include": [
-  #           "slsa_provenance_available"
-  #         ]
-  #       }
-  #     }
-  #     ```
-  #   When version 0.1 of the task named "verify-enterprise-contract" is run with parameters:
-  #     | IMAGES               | {"components": [{"containerImage": "quay.io/hacbs-contract-demo/golden-container@sha256:e76a4ae9dd8a52a0d191fd34ca133af5b4f2609536d32200a4a40a09fdc93a0d"}]} |
-  #     | POLICY_CONFIGURATION | ${NAMESPACE}/${POLICY_NAME}                                                                                                                                  |
-  #     | STRICT               | true                                                                                                                                                         |
-  #   Then the task should succeed
+  Scenario: Golden container image
+    Given a working namespace
+    Given a cluster policy with content:
+      ```
+      {
+        "publicKey": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAERhr8Zj4dZW67zucg8fDr11M4lmRp\nzN6SIcIjkvH39siYg1DkCoa2h2xMUZ10ecbM3/ECqvBV55YwQ2rcIEa7XQ==\n-----END PUBLIC KEY-----",
+        "sources": [
+          {
+            "policy": [
+              "github.com/hacbs-contract/ec-policies//policy"
+            ]
+          }
+        ],
+        "configuration": {
+          "include": [
+            "slsa_provenance_available"
+          ]
+        }
+      }
+      ```
+    When version 0.1 of the task named "verify-enterprise-contract" is run with parameters:
+      | IMAGES               | {"components": [{"containerImage": "quay.io/hacbs-contract-demo/golden-container@sha256:e76a4ae9dd8a52a0d191fd34ca133af5b4f2609536d32200a4a40a09fdc93a0d"}]} |
+      | POLICY_CONFIGURATION | ${NAMESPACE}/${POLICY_NAME}                                                                                                                                  |
+      | STRICT               | true                                                                                                                                                         |
+    Then the task should succeed
 
   Scenario: Verifying a simple task definition
     Given a working namespace
-  
+
     When version 0.1 of the task named "verify-definition" with workspace "output" is run with parameters:
       | DEFINITION    | {"kind": "Task"}                                        |
       | POLICY_SOURCE | git::github.com/hacbs-contract/ec-policies//policy/task |
$ make focus-acceptance 
❱ dist/ec_linux_amd64
❱ build
❱ build
❱ build
❱ focus-acceptance
ok  	github.com/hacbs-contract/ec-cli/acceptance	212.674s
```